### DISCUSSION TITLE
Is there a LineColumn or anything similar?

### DISCUSSION THREAD

**Discussion by jdion84** [filamentphp/filament#7791] [Help] [answered]
I have 5 different columns I would like to put side-by-side into a single column:

For example:

```php
LineColumn::make('statuses')
    ->columns([
        TextColumn::make('assigns_count')
            ->icon('heroicon-o-paper-airplane')
            ->counts('assigns'),

        TextColumn::make('returns_count')
            ->icon('heroicon-o-receipt-refund')
            ->counts('returns'),

        IconColumn::make('confirmed')
            ->icon('heroicon-o-device-phone-mobile')
            ->trueColor('success')
            ->falseColor('danger'),

        IconColumn::make('validated')
            ->icon('heroicon-o-check-circle')
            ->trueColor('success')
            ->falseColor('danger'),

        Tables\Columns\IconColumn::make('sent_to_call_centre')
            ->icon('heroicon-o-device-phone-arrow-up-right')
            ->trueColor('success')
            ->falseColor('danger'),
    ])
```

I've read the docs multiple times and can't seem to find this, so if I missed it, I apologize in advance.

I know I can just use a custom view, but I was wondering if this is something I can do natively with filament.

Thanks.

**Comment by leandrocfe** [comment]
Custom column or `formatStateUsing()` I think

**Reply by jdion84** [reply]
that method does not appear to accept actual columns as the value.

**Comment by mohamedsabil83** [comment, marked as answer]
If you mean as a layout, I guess you can use **Stack**/**Split** and configure it as you need

**Reply by jdion84** [reply]
I know I can use a grid/split but that seems to take over the entire table layout. I'm just trying to do this for a single column.
